packages/techdocs-common/src/stages/publish/azureBlobStorage.ts

```diff
@@ -82,10 +82,16 @@ export class AzureBlobStoragePublish implements PublisherBase {
     await storageClient
       .getContainerClient(containerName)
       .getProperties()
-      .then(() => {
-        logger.info(
-          `Successfully connected to the Azure Blob Storage container ${containerName}.`,
-        );
+      .then(result => {
+        if (result?._response?.status >= 400) {
+          logger.error(
+            'Could not read Azure Blob Storage container properties',
+          );
```

Comment:
Let's throw an error from this block. We don't want to initialize an Azure publisher which is pretty much useless without Auth 😅

Reply:
Done in https://github.com/backstage/backstage/pull/4539/commits/4de00ead3887a1dd8307745b1ab46d3e40a1f58a